lib/dotcom_web/components/badges.ex

```diff
@@ -0,0 +1,36 @@
+defmodule DotcomWeb.Components.Badges do
+  use Phoenix.Component
+
+  attr :class, :string, default: ""
+  slot :inner_block
+
+  def circle_icon(assigns) do
+    ~H"""
+    <span class={[
+      "select-none",
+      "h-6",
+      "w-6",
+      "rounded-full",
+      "font-bold",
+      "text-white",
+      "flex",
+      "justify-center",
+      "items-center",
+      @class
+    ]}>
+      <%= render_slot(@inner_block) %>
+    </span>
+    """
+  end
+
+  attr :class, :string, default: ""
+  slot :inner_block
+
+  def rect_icon(assigns) do
```

Comment:
Or this.

Reply:
Historical context:

I had actually built this before you added the badges to metro, so the path of least resistance was to leave them. I did try to migrate to using metro, but ran into a bug where they came out looking wrong. I actually did leave a screenshot in the PR description. 

I'm willing to spend a small amount of time trying to get metro to work with this, but depending on how long that takes, it might make more sense to do that as a follow-up PR, rather than blocking this PR on it.

Reply:
If you have problems integrating metro, let me know and I can probably figure it out quickly. Merging a PR with stopgaps that we just have to change later doesn't really save us anything.

Reply:
```
<.badge type="square" color="yellow" class="text-yellow-800 rounded-sm">66</.badge>
```
Fix the override for `p` in bootstrap's `_reboot.css`.

![Screenshot Capture - 2024-10-29 - 15-40-45](https://github.com/user-attachments/assets/2d625ef7-4c8e-4225-83ca-24aae20d3821)

Reply:
Beware! That might very well have consequence on other pages across Dotcom using `<p>`

Reply:
```
<.badge type="square" color="yellow" class="mbta-metro-badge text-yellow-800 rounded-sm">66</.badge>
```

```
.mbta-metro-badge p {
  margin: 0;
}
```

Reply:
Could also change it to a `span` inside the badge.